src/alt/index.js

```diff
@@ -138,10 +145,28 @@ class Alt {
       assign(actions, ActionsClass)
     }
 
+    this.actions[key] = this.actions[key] || {}
+
     return Object.keys(actions).reduce((obj, action) => {
-      obj[action] = this.createAction(`${key}#${action}`, actions[action], obj)
+      if (typeof actions[action] !== 'function') {
+        return obj
+      }
+
+      // create the action
+      const name = `${key}#${action}`
+      obj[action] = this.createAction(name, actions[action], obj)
+
+      // remove the action referenced in single
```

Comment:
global feels right now that I think about it again. It's global to your alt instance.